lighthouse-core/lib/url-shim.js

```diff
@@ -10,6 +10,7 @@
  */
 
 const {Util} = require('../util-commonjs.js');
+const LHError = require('../lib/lh-error.js');
```

Comment:
So turns out this creates a circular dependency that messes with our DT bundle. `url-shim.js` might not be the best place for this @brendankenny.

Reply:
`runner-helpers.js` maybe?